Kitodo/src/main/java/org/kitodo/services/data/TaskService.java

```diff
@@ -87,67 +88,8 @@ public static TaskService getInstance() {
     }
 
     /**
-     * Creates and returns a query to retrieve tasks for which the currently logged
-     * in user is eligible.
-     *
-     * @param user
-     *            currently logged in user
-     * @return query to retrieve tasks for which the user eligible.
-     */
-    private BoolQueryBuilder createUserTaskQuery(User user) {
```

Comment:
Why are you removing this which was added a few days ago by @oliver-stoehr and @solth ? Maybe a wrong rebase?